packages/apollo-forest-run/src/cache/env.ts

```diff
@@ -37,6 +38,8 @@ export function createCacheEnvironment(config?: CacheConfig): CacheEnv {
     autoEvict: config?.autoEvict ?? true,
     nonEvictableQueries: config?.nonEvictableQueries ?? new Set(),
     maxOperationCount: config?.maxOperationCount ?? 1000,
+    logger: config && "logger" in config ? config.logger : logger,
```

Comment:
question: based on this the logger should never be undefined. Why in functions props the logger is possibly undefined?